api/logging/v1/output_types.go

```diff
@@ -320,7 +320,8 @@ type Http struct {
 	// +kubebuilder:validation:Enum:=opentelemetry;viaq
 	// +kubebuilder:default:viaq
 	// +optional
-	Schema string `json:"schema,omitempty"`
+	// +operator-sdk:csv:customresourcedefinitions:xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
```

Comment:
Curious, why not remove this as well?

Reply:
Attempting to not remove anything so we can reuse it when available.  This should make it so its not public though thinking about it, did this already exist in 5.8?

Reply:
Yes this field did exist in 5.8. I am not inclined to remove this because of what you stated.